lib/interactive_viewer.ml

```diff
@@ -47,7 +47,14 @@ let ui_of_operation operation =
       Ui.hcat
         [ W.string "Modification of "; W.string ~attr:blue_bold_attr path ]
 
-let string_of_hunk = Format.asprintf "%a" Patch.pp_hunk
+let string_of_hunk hunk =
```

Comment:
Small nitpick: this function is not anymore converting to a string but to a UI element, so the `ui_of_hunk` name would make more sense!